docs/ancestry.md

```diff
@@ -958,59 +958,179 @@ not ancestral to these census nodes.
 
 ## Specifying the initial state
 
+By default `msprime` simulations are initialised by specifying a set of
+{ref}`samples<sec_ancestry_samples>`,
+which sets up the simulation with segments of ancestral material covering the
+whole sequence. Internally, this initial state is implemented by
+setting up a {class}`tskit.TableCollection` which contains the
+{ref}`population<tskit:sec_population_table_definition>`,
+{ref}`node<tskit:sec_node_table_definition>`,
+and {ref}`individual<tskit:sec_individual_table_definition>` information.
+The low-level simulation code then reads these tables and sets up the
+simulation accordingly. We can see what this initial state looks like
+by setting up a simple simulation and stopping it running immediately:
+
+```{code-cell}
+initial_ts = msprime.sim_ancestry(1, end_time=0, random_seed=2)
+initial_ts
+```
+We can see that the returned tree sequence has one population,
+two sample nodes and one individual (since sample individuals are
+{ref}`diploid<sec_ancestry_ploidy>` by default). There are no
+{ref}`edges<tskit:sec_edge_table_definition>` because the simulation
+didn't run for any period of time: we're essentially just returning
+the initial state used to start the simulation.
+
+Then, running a simulation in which the ``initial_state`` is equal to
+``initial_ts`` is *precisely* the same as running the simulation
+with the same arguments as we used above:
```

Comment:
but, this is only true because all the all the arguments above were set to defaults, right? - saying it this way might lead someone to think that all other arguments to `sim_ancestry( )` were somehow recorded in the tables and would be passed through?

Reply:
Good point - I've added a warning block.